website/docusaurus.config.js

```diff
@@ -0,0 +1,180 @@
+  organizationName: "typescript-cheatsheets",
+
+  // Add custom scripts here that would be placed in <script> tags.
+  // scripts: ["https://buttons.github.io/buttons.js"],
+
+  presets: [
+    [
+      "@docusaurus/preset-classic",
+      {
+        theme: {
+          customCss: require.resolve("./src/css/custom.css"),
+        },
+        docs: {
+          // Docs folder path relative to website dir.
+          path: "../docs",
+          // Sidebars file relative to website dir.
+          sidebarPath: require.resolve("./sidebars.json"),
+        },
+        // ...
+      },
+    ],
+  ],
+
+  themeConfig: {
+    defaultDarkMode: true,
+
+    image:
+      "https://user-images.githubusercontent.com/6764957/53868378-2b51fc80-3fb3-11e9-9cee-0277efe8a927.png",
+
+    // Equivalent to `docsSideNavCollapsible`.
+    // sidebarCollapsible: false,
+
+    prism: {
+      defaultLanguage: "typescript",
+    },
+
+    navbar: {
+      title: "React TypeScript Cheatsheet",
+      logo: {
+        alt: "Logo",
+        src: "img/icon.png",
+      },
+      links: [
+        {
+          to: setupDoc,
+          label: "Docs",
+          position: "right",
+        },
+        {
+          to: "help",
+          label: "Help",
+          position: "right",
+        },
+        // {to: 'blog', label: 'Blog', position: 'right'},
+      ],
+    },
+
+    footer: {
+      style: "dark",
+      logo: {
+        alt: "TypeScript Cheatsheets Logo",
+        src: "img/icon.png",
+        maxWidth: 128,
+        style: { maxWidth: 128, maxHeight: 128 },
+      },
+      copyright: `Copyright © ${new Date().getFullYear()} TypeScript Cheatsheets`,
+      links: [
+        {
+          title: "Docs",
+          items: [
+            {
+              label: "Introduction",
+              to: setupDoc,
+            },
+            {
+              label: "High Order Component (HOC)",
+              to: "docs/hoc/intro",
+            },
+            {
+              label: "Advanced Guides",
+              to: "docs/advanced/intro",
+            },
+            {
+              label: "Migrating",
+              to: "docs/migration/intro",
+            },
+          ],
+        },
+        {
+          title: "Community",
+          items: [
+            {
+              label: "Stack Overflow",
+              href: "https://stackoverflow.com/questions/tagged/typescript",
+            },
+            {
+              label: "User Showcase",
+              to: "users",
+            },
+            {
+              label: "Help",
+              to: "help",
+            },
+          ],
+        },
+        {
+          title: "More",
+          items: [
+            {
+              label: "GitHub",
+              href:
+                "https://github.com/typescript-cheatsheets/react-typescript-cheatsheet",
+            },
+            /*
+            {
+              label: "Star",
+              class: "footer__link-item github-button",
```

Comment:
We didn't document the footer well, but you could do this for the GitHub star button instead - https://github.com/facebook/docusaurus/blob/master/website/docusaurus.config.js#L194-L197

Reply:
this would be nice! not sure where to pull it from but can investigate later